resolwe/flow/managers/__init__.py

```diff
@@ -1,6 +1,30 @@
-"""Workflow workload managers."""
+""".. Ignore pydocstyle D400.
+
+=============
+Flow Managers
+=============
+
+Workflow workload managers.
+
+.. data:: manager
+
+    The global manager instance.
+
+    :type: :class:`~resolwe.flow.managers.dispatcher.Manager`
+
+.. automodule:: resolwe.flow.managers.dispatcher
```

Comment:
I would change the title of the `resolwe.flow.managers.dispatcher`'s docstring to _Dispatcher_.